docs/oracle/oracle-election-example.md

```diff
@@ -0,0 +1,111 @@
+---
+id: oracle-election-example
+title: Election Example
+---
+
+## Requirements for example
+
+You need to have a fully built oracle server. You can follow [this guide](build-oracle-server.md) to do this.
+
+You will also need a the `bitcoin-s-cli` command line tool to interact with the server.
+You can find how to build this [here](../applications/cli.md)
+
+After building the oracle server, you will need to start it with
+
+```
+./app/oracle-server/target/universal/stage/bin/bitcoin-s-oracle-server
+```
+
+## US 2020 Election
+
+In 2020, the United States held a presidential election. Nicolas Dorier and Chris Stewart
+wanted to do a DLC based on the outcome of this presidential election.
```

Comment:
> Nicolas Dorier and Chris Stewart wanted to do a DLC based on the outcome of this presidential election.

imo this should removed since the oracle does not have knowledge of their users

Reply:
The purpose is to create a story so users have a frame of reference of what the purpose of the documentation is.

Reply:
Still, the story shouldn't have to involve explict users. Could do something like

> In 2020, the United States held a presidential election. People want to do a DLC based on the outcome, so you decide to be their oracle and attest to the winner of the election.